docs/reference/query-dsl/semantic-query.asciidoc

```diff
@@ -40,9 +40,209 @@ The `semantic_text` field to perform the query on.
 (Required, string)
 The query text to be searched for on the field.
 
+`inner_hits`::
+(Optional, object)
+Retrieves the specific passages that match the query.
+See <<semantic-query-passage-ranking, Passage ranking with the `semantic` query>> for more information.
++
+.Properties of `inner_hits`
+[%collapsible%open]
```

Comment:
I don't think we usually collapse query DSL properties, based on looking at a few other pages. It's probably not a huge deal but inconsistent with pages like `knn` and `text_expansion` in the same grouping. Those pages also aren't using the `.Properties` syntax. I'll defer to docs experts on which way is "right" 🙂

Reply:
I like the look of the collapsible blocks better personally, that's why I went with it :)

@leemthompo Any guidance you can offer here?